app/src/main/res/values/strings.xml

```diff
@@ -47,4 +48,19 @@
     <string name="order_range">Order Range</string>
     <string name="tickets_available">Tickets Available</string>
 
+    <!--suggestion mail details-->
+    <string name="emailInfo">List out your proposed Bug(s)/Feature(s)</string>
+    <string name="emailSubject">Improvement Suggestion for Open Event General App</string>
+    <string name="testEmailId">open.test24@gmail.com</string>
```

Comment:
Why this email? eventyay has its own email

Reply:
Wasn't this intended for reporting bugs or features in the app? I thought that it would be better to receive emails directly about the bugs to the developers mail

Reply:
Please  go through reviews of other PRs as well https://github.com/fossasia/open-event-android/pull/150
and to know why this email was made you can have a look at the conversation here https://gitter.im/fossasia/open-event-android

Reply:
@nikit19 Bad misunderstanding! Thanks :+1: